dm/loader/util.go

```diff
@@ -142,14 +154,18 @@ func getMydumpMetadata(cli *clientv3.Client, cfg *config.SubTaskConfig, workerNa
 }
 
 // cleanDumpFiles is called when finish restoring data, to clean useless files.
-func cleanDumpFiles(cfg *config.SubTaskConfig) {
+func cleanDumpFiles(ctx context.Context, cfg *config.SubTaskConfig) {
 	log.L().Info("clean dump files")
 	if cfg.Mode == config.ModeFull {
 		// in full-mode all files won't be need in the future
-		if err := os.RemoveAll(cfg.Dir); err != nil {
+		if err := exstorage.RemoveAll(ctx, cfg.Dir, nil); err != nil {
 			log.L().Warn("error when remove loaded dump folder", zap.String("data folder", cfg.Dir), zap.Error(err))
 		}
 	} else {
+		if exstorage.IsS3Path(cfg.Dir) {
+			// s3 no need immediately remove
```

Comment:
so when will this file will be removed?

Reply:
also please left a warning log here

Reply:
ok, dumpfile will be removeall at syncer, https://github.com/pingcap/tiflow/pull/4361/files#diff-d1f25eda3f1652cbf880bc0ff9477335e6ed3d9aea0805fa83d43b2373484e6aR1576